src/NuGet.Core/NuGet.CommandLine.XPlat/ReportRenderers/Interfaces/IReportRenderer.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Why have a directory with only a single file? Again the file's int he same namespace as everything else, so I don't understand the intent in organization of the code.

Reply:
Sure, moved to next to other files. 
Sorry, I assumed using namespace `NuGet.CommandLine.XPlat` was our convention, fixed it.